srcpkgs/python3-bibtexparser/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'python3-bibtexparser'
 pkgname=python3-bibtexparser
-version=1.1.0
-revision=5
+version=1.2.0
+revision=1
 wrksrc="bibtexparser-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
```

Comment:
```suggestion
hostmakedepends="python3-setuptools python3-pip python3-wheel"
```

Append these two packages.

Reply:
No, this is wrong, pip should never be in any dependencies.

The check error says it needs python3-future in checkdepends.
```
error: Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmphjmnl4oi', '--quiet', 'future>=0.16.0']' returned non-zero exit status 1.
```

Reply:
@paper42, according to what you just wrote, I suppose you meant the following:

```
hostmakedepends="python3-setuptools python3-wheel"
depends="python3-parsing"
checkdepends="python3-future"
```

However, it still does not seem to be enough:

```
=> python3-bibtexparser-1.2.0_1: running do_check ...
/usr/lib/python3.10/distutils/dist.py:274: UserWarning: Unknown distribution option: 'extra_requires'
  warnings.warn(msg)
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
/usr/bin/python3: No module named pip
error: Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmpfxlnb_mp', '--quiet', 'pyparsing>=2.0.3']' returned non-zero exit status 1.
=> ERROR: python3-bibtexparser-1.2.0_1: do_check: 'python3 setup.py ${make_check_target} ${make_check_args}' exited with 1
=> ERROR:   in do_check() at common/build-style/python3-module.sh:40
```

Reply:
```
hostmakedepends="python3-setuptools"
depends="python3-parsing"
checkdepends="python3-future $depends"
```
It doesn't actually run any tests because the pypi tarball doesn't contain them and we can't switch the tarball source to github because they didn't publish 1.2.0 there: https://github.com/sciunto-org/python-bibtexparser/issues/287. But that shouldn't be a blocker.

Reply:
Thanks @paper42 @dotnetfox for the suggestion. CI is all green now after applying the new changes :)